core/src/main/java/nl/nn/adapterframework/monitoring/SenderMonitorAdapter.java

```diff
@@ -75,7 +75,7 @@ public void fireEvent(String monitorName, EventType eventType, Severity severity
 				session.put(PipeLineSession.ORIGINAL_MESSAGE_KEY, newMessage);
 				session.scheduleCloseOnSessionExit(newMessage, "Event fired by "+ monitorName);
 			}
-			getSender().sendMessageOrThrow(new Message(makeXml(monitorName, eventType, severity, eventCode, event)), session);
+			getSender().sendMessageOrThrow(new Message(makeXml(monitorName, eventType, severity, eventCode, event)), session); // close() disables unit testing Message result
```

Comment:
@tnleeuw Here it only gives issues for the unit tests. I think for the normal operations, it works fine.

Reply:
> @tnleeuw Here it only gives issues for the unit tests. I think for the normal operations, it works fine.

Correct, because the unit tests use a capture at time of sending the message and now that is closed, whereas before it wasn't.
(Result that is closed and discarded happens to be the same as the input which is verified in the tests).

Can be fixed by capturing instead a copy of the message in the unit tests, but we can also leave it like it is for now and add `close()` at a later point.